src/core/plugins/core-fingerpoint/server/index.ts

```diff
@@ -0,0 +1,8 @@
+import * as alt from 'alt-server';
+import { PluginSystem } from '../../../server/systems/plugins';
```

Comment:
You can actually remove this entire file. It's only necessary to register a plugin on server-side.